src/Microsoft.AspNetCore.SignalR/SignalRAppBuilderExtensions.cs

```diff
@@ -32,9 +32,14 @@ public HubRouteBuilder(SocketRouteBuilder routes)
             _routes = routes;
         }
 
-        public void MapHub<THub>(string path) where THub : Hub
+        public void MapHub<THub>(string path) where THub : Hub<IClientProxy>
```

Comment:
This is actually needed if you try to map a hub the derives from `Hub<IClientProxy>` instead of `Hub`.  I have `HubWithPresence : HubWithPresence<IClientProxy>` and `HubWithPresence<TClient> : Hub<TClient>` and it is causing an eror: `There is no implicit reference conversion from 'ChatSample.Hubs.Chat' to 'Microsoft.AspNetCore.SignalR.Hub'.` I can create `HubWithPresence : Hub` and it will work but the other should work as well.